modules/haskell-integration-tests/mu-haskell-client-server/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM fpco/stack-build:lts-14.27 as build
```

Comment:
Maybe you can use here the latest lts we just migrated to!
```suggestion
FROM fpco/stack-build:lts-15.5 as build
```

Reply:
The tests are currently using mu-haskell 0.2, which uses `lts-14.27`. Maybe it makes sense to upgrade the LTS at the same time we upgrade the tests to mu-haskell 0.3?

Reply:
That makes sense! 😄